openapi_python_client/schema/openapi_schema_pydantic/path_item.py

```diff
@@ -1,11 +1,14 @@
-from typing import Optional, Union
+from typing import TYPE_CHECKING, Optional, Union
 
 from pydantic import BaseModel, ConfigDict, Field
 
 from .parameter import Parameter
 from .reference import Reference
 from .server import Server
 
+if TYPE_CHECKING:
+    from .operation import Operation
```

Comment:
```suggestion
    from .operation import Operation  # pragma: no cover
```

Otherwise the coverage check in CI will fail.